Lib/test/libregrtest/pgo.py

```diff
@@ -35,6 +35,7 @@
     'test_memoryview',
     'test_operator',
     'test_ordered_dict',
+    'test_patma',
```

Comment:
FYI, I'm on the fence about this... I only added it because we now have a bunch of new code in the interpreter core with zero PGO coverage.

I might run the pyperformance suite tonight with and without `test_patma` included here, and just revert this if there are any performance regressions.